float64 x
float64 y
bool object_found
bool classifier
bool led
